term_proposal/YSHsieh7777/README.md

```diff
@@ -0,0 +1,57 @@
+# Term Proposal - Crowd Evacuation Simulation
+
+## Table of Contents
+
+ - [Motivation](#motivation)
+ - [Ideas](#ideas)
+ - [To-Do List](#to-do-list)
+ - [Example](#example)
+ - [Reference](#reference)
+
+## Motivation
+
+People often fall into **panic** when they meet dangers like fire or earthquake. Losing the ability to thinking and acting calmly make them lose the opportunity of being out of danger. If we can **simulate the situation** that crowd evacuate from indoor to outdoor when breaking out of the fire, then we can schedule the escape route more smoothly to prevent disaster.
+
+## Ideas
+
+### Behaviors of Human
+ - Simulate the **flocking behavior** of humans.
+ - Simulate the **herd behavior** of humans.
+ - Simulate the **panic behavior** of humans when encountering dangers.
+ - All behaviors listed above are factors that will affect people's actions. Each factor will be assigned a **floating weight** so that each factor will have different influences under different situations.
+
+### Targets of Program
+ - **Simulate and visualize** the situation that crowd evacuate from indoor to outdoor.
+ - Take **human behaviors** into consideration to determine crowd's action.
+ - Let user **design** the indoor traffic flow.
+ - Calculate the amount of time that people evacuate from indoor to outdoor.
+ - Calculate the number of people that die in the fire. 
+ - Store the **best** traffic flow.
+
+## To-Do List
+
+ - [x] Find Term Project topic. ( 1 week )
+ - [x] Collect information on that topic. ( 1-2 weeks )
+ - [x] Determine the targets of the project. ( 1 week )
+ - [ ] Plan and build the project structure. ( 1-2 week )
+ - [ ] Coding and debugging. ( 4-6 weeks )
+ - [ ] Refine features and codes. ( 1-2 weeks ) 
+ - [ ] Prepare the presentation. ( 1 week )
+
+## Example
```

Comment:
You should describe how you are architecting the system.  Nothing may be crystal clear at this point, but attempting to specify will help you down the road.

Include things like language, data structure, system facilities, when they apply.